### ISSUE TITLE
Trinity does not launch on Motorola and LG devices

### ISSUE THREAD

**Issue by rajivshah3** [iotaledger/trinity-wallet#1135]
### Bug description
Trinity crashes as soon as the user launches it on some Motorola and LG devices

### Environment Specs
This crash has been reported on:

- Motorola Moto G3 - Android 6.0.1
- Motorola Moto G4 - Android 6.0.1 (via Google Play pre-launch testing)
- LG K3 2017 - Android 6.0.1 (via Google Play pre-launch testing)

What build of Trinity are you running? Please provide the full version (you can find it in Settings > About Trinity).

- Trinity version: v0.6.2 (44) and v0.6.2 (49)

### Steps To Reproduce
1. Open Trinity on an affected device

### Expected behaviour
App should not crash

### Actual behaviour
App crashes

### Errors

```
FATAL EXCEPTION: main
Process: com.iota.trinity, PID: 14504
java.lang.UnsatisfiedLinkError: dlopen failed: cannot locate symbol "__aeabi_memcpy" referenced by "/data/app/com.iota.trinity-1/lib/arm/libdummy.so"...
	at java.lang.Runtime.loadLibrary(Runtime.java:372)
	at java.lang.System.loadLibrary(System.java:1076)
	at com.iota.trinity.MainApplication.attachBaseContext(MainApplication.java:53)
	at android.app.Application.attach(Application.java:187)
	at android.app.Instrumentation.newApplication(Instrumentation.java:1020)
	at android.app.Instrumentation.newApplication(Instrumentation.java:1004)
	at android.app.LoadedApk.makeApplication(LoadedApk.java:573)
	at android.app.ActivityThread.handleBindApplication(ActivityThread.java:4717)
	at android.app.ActivityThread.-wrap1(ActivityThread.java)
	at android.app.ActivityThread$H.handleMessage(ActivityThread.java:1418)
	at android.os.Handler.dispatchMessage(Handler.java:102)
	at android.os.Looper.loop(Looper.java:148)
	at android.app.ActivityThread.main(ActivityThread.java:5459)
	at java.lang.reflect.Method.invoke(Native Method)
	at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:728)
	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:618)
```

**Comment by rajivshah3** [comment]
Actually, this might be an Android 6 issue, not a Motorola/LG issue

**Comment by rajivshah3** [comment]
@thibault-martinez do you know why this might be happening? If I run `objdump -t` on the `libdummy.so` in our `develop` branch (which was built a long time ago) it looks different from the one that I built locally from the latest `entangled` commit:

```bash
objdump -t /Users/rajiv/wallet/src/mobile/android/app/src/main/jniLibs/armeabi-v7a/libdummy.so | grep mem

00000000 g     F *UND*		 00000000 __aeabi_memcpy
00000000 g     F *UND*		 00000000 __aeabi_memclr
00000000 g     F *UND*		 00000000 __aeabi_memclr8
00000000 g     F *UND*		 00000000 memcpy
00000000 g     F *UND*		 00000000 __aeabi_memcpy8
00000000 g     F *UND*		 00000000 __aeabi_memclr4
00000000 g     F *UND*		 00000000 __aeabi_memcpy4
00000000 g     F *UND*		 00000000 memchr
```

```bash
objdump -t dummyLibs/lib/armeabi-v7a/libdummy.so | grep mem

00000000 g     F *UND*		 00000000 memcpy
00000000 g     F *UND*		 00000000 memset
00000000 g     F *UND*		 00000000 memcmp
00000000 g     F *UND*		 00000000 __aeabi_memclr
00000000 g     F *UND*		 00000000 __aeabi_memcpy
00000000 g     F *UND*		 00000000 __aeabi_memset
00000000 g     F *UND*		 00000000 __aeabi_memclr8
```

Edit: I hope I didn't cause this with https://github.com/iotaledger/entangled/commit/3a392503054b3c432031125630b6af788f606480 😅 but https://github.com/android-ndk/ndk/issues/126 seems to imply that using an earlier version helps

**Comment by rajivshah3** [comment]
Fixed in #1138
